odk1-src/form-language.rst

```diff
@@ -101,6 +102,11 @@ For example:
   the non-specific version of that column
   will be treated as if it were a separate language.
   (The :menuselection:`Change Language` menu will list it as :guilabel:`Default`.)
+  
+  To avoid this, all columns that can be made multi-lingual, need to be created 
```

Comment:
I /think/ there shouldn't be a comma after "multi-lingual" because the "all columns" clause is the subject of "need to".

I think having the example XLSForm that you included in a comment would be helpful. I agree the screenshots are a lot so maybe just the table would be good.

Reply:
> there shouldn't be a comma after "multi-lingual"
agree

> the example XLSForm
agree